contrib/tracking.cpp

```diff
@@ -0,0 +1,87 @@
+#include "tracking.h"
+#include <opencv2/opencv.hpp>
+
+
+bool Tracker_Init(Tracker self, Mat image, Rect boundingBox) {
+    cv::Rect2d c_boundingBox(boundingBox.x, boundingBox.y, boundingBox.width, boundingBox.height);
+
+    bool ret = (*self)->init(*image, c_boundingBox);
+    return ret;
+}
+
+
```

Comment:
This comment applies to the whole file: I think we should only have one blank line between the functions.